css/exercise_two/index.html

```diff
@@ -0,0 +1,361 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Exercise two CSS</title>
+  <link rel="stylesheet" type="text/css" href="styles/styles.css">
+  <script src="node_modules/bootstrap/dist/js/bootstrap.bundle.min.js" crossorigin="anonymous"></script>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400;500&display=swap" rel="stylesheet">
+</head>
+<body>
+  <nav class="container align-items-center d-flex justify-content-between mb-7 pt-5">
```

Comment:
just a comment for the readiness of the classes, it's always better to have a parent class like d-flex first,
align-items-center and justify-content-between both are dependent on having d-flex.

it won't affect the result, but someone else that reads this will understand better